### PR TITLE
TC_05.004.01 | Multibranch Pipeline > Disable Multibranch Pipeline >Check to see if you can disable the multi-branch pipeline on the configuration page of the created folder

### PR DESCRIPTION
**Pull request by kristinasharapova**
https://trello.com/c/RNyqoRUE/725-tc0500401-item-sub-item-us-title

### CODE REVIEW

**Review by tereshenkov29 — APPROVED** [review]

**Review by judmi — COMMENTED** [review]
Incorrect class name. We name classes according to the instances that are being tested in them (e.g. PipelineTest, FreestyleProjectTest etc). We do not create classes to test separate functionality of an item. if Can you rename it please?

File: src/test/java/school/redrover/DisableMultibranchPipelineTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+public class DisableMultibranchPipelineTest extends BaseTest {
```
Comment (EvgenyJavaTest) [inline src/test/java/school/redrover/DisableMultibranchPipelineTest.java:9]: In my mind, the name of the class should point to the object, and not to actions with it (ex. MultibranchPipelineTest)

File: src/test/java/school/redrover/DisableMultibranchPipelineTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        getDriver().findElement(By.id("ok-button")).click();
+
+    }
+
+    @Test
+    public void testDisableMultibranchPipeline() throws InterruptedException {
+        createMultibranchPipeline("Test_Folder");
+
+        getDriver().findElement(By.xpath("//span[@id='toggle-switch-enable-disable-project']/label")).click();
+        Thread.sleep(2000);
```
Comment (EvgenyJavaTest) [inline src/test/java/school/redrover/DisableMultibranchPipelineTest.java:30]: We're not recommended to use Thread.sleep(), as it slows down the test run. It is better to use explicit waits.

**Review by SafronovaOlga — APPROVED** [review]

**Comment by Esmartya** [comment]
@kristinasharapova could you squash and merge it, please? I have 2 tests inked with your cards TC_05.004.01, TC_05.004.02 ))))

**Pull request merged**